frontend/scripts/replace-variable.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+ORIGINAL=$1
+REPLACEMENT=$2
+
+if [ "${ORIGINAL}" = "${REPLACEMENT}" ]; then
+    echo "Replacement is the same, skipping.."
+    exit 0
+fi
+
+echo "Replacing pre-baked value.."
+
+find /app/public /app/.next -type f ! -name "*.png" ! -name "*.svg" ! -name "*.gif" ! -name "*.jpg" ! -name "*.jpeg" ! -name "*.ico" |
```

Comment:
Instead of doing if file name not a certain extension, do you think it would be better to just scope to only. tsx and .js files? I don't think we will have any other file types that will need to be executed since this is a JS project. What do you think?

Reply:
Sounds good, let me know your finding. Also what performance did you notice when doing the text replace when a override is provided? 

Sidenote, i have also added the `PUBLIC_POSTHOG_API_KEY` to our secrets so that is good to go once we push this one out.

Reply:
> let me know your finding

I found out that just `.js` files 😄. So I updated this PR to limit replacement to only these.

> what performance did you notice when doing the text replace

With the last replace script, it took ~2s for the frontend to boot up. I don't have the timings of the actual replace though 😅. My latest change is limited to only `.js` files so definitely a lot faster now I hope ✌️

> i have also added the PUBLIC_POSTHOG_API_KEY to our secrets

Cool! Thanks!